src/templates/commit_page.handlebars

```diff
@@ -368,6 +362,18 @@
                             </span>
                         </li>
                         <li>
+                            <i class="section-group__item icon-check{{#is_false original.src_options.is_fanon}}-empty{{/is_false}}" data-field="is_fanon"/>
+                            <span class="check-txt">
+                                Is Fanon
```

Comment:
Sounds like these are mediawiki source that can sometimes have articles on real-world topics.  We want to be careful about showing them.  I didn't pick the name :smile: